tests/performance/test_metrics.py

```diff
@@ -10,8 +10,8 @@
 from aws_lambda_powertools.metrics import metrics as metrics_global
 
 # adjusted for slower machines in CI too
-METRICS_VALIDATION_SLA: float = 0.002
-METRICS_SERIALIZATION_SLA: float = 0.002
+METRICS_VALIDATION_SLA: float = 0.003
+METRICS_SERIALIZATION_SLA: float = 0.003
```

Comment:
Once I added python 3.10 the performance got slightly worse :(

Reply:
Ruben, I reverted this change. Looks like it was a one-time glitch.